### DISCUSSION TITLE
Solana DeFi Trading Platform Mango Markets Loses $100M in Hack

### DISCUSSION THREAD

**Discussion by danwan1027** [1101-GS4719-C/Python-Community-GS4719-C#767] [General]
才經歷了 BNB chain 的跨鏈橋被駭，不到一個禮拜，又傳出 mango market 被駭的新聞，mango 是 Solana 公鏈上的一個借貸項目，大家在選擇投資工具時，一定要做足功課且不要把雞蛋都放在同一個籃子裡。
附上原文：https://decrypt.co/111727/solana-defi-trading-platform-mango-markets-loses-100m-in-hack

**Comment by a45320614** [comment]
好想學習區塊鍊相關的知識喔，要怎麼樣才能在中央學習相關的知識呢

**Reply by williamyen0623** [reply]
加入區塊鏈研究社或是修葉羅堯老師區塊鏈的課～

**Reply by Yunhans** [reply]
感謝分享

**Comment by williamyen0623** [comment]
斥責那些駭客也不是辦法，希望業界大老能盡快解決DeFi的種種問題，讓DeFi項目能發展起來！

**Comment by Yunhans** [comment]
我還以為區塊鏈是沒有辦法被駭入的，好扯喔

**Comment by Pietra1226** [comment]
我也以為區塊鍊是幾乎不可能被駭入的，好強

**Comment by StevenWanglolz** [comment]
區塊鏈這種東西真的要謹慎投資，因為沒有受到法律的保障
